<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>UserMenu</class>
  <widget class="QDialog" name="UserMenu">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Меню пользователя</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QLabel" name="label">
          <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Меню&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btn_scripts">
          <property name="text">
            <string>Записи склада</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btn_balance">
          <property name="text">
            <string>Баланс ингредиентов</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btn_close">
          <property name="text">
            <string>Закрыть</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>